cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/store/MetadataStoreTest.java

```diff
@@ -337,6 +342,7 @@ public void testSearchWeight() throws Exception {
     Assert.assertEquals(expected, actual);
 
     actual = Lists.newArrayList(store.searchMetadata("ns1", "value1 sValue*"));
+    List<MetadataSearchResultRecord> actualSearchAll = Lists.newArrayList(store.searchMetadata("ns1", "*"));
```

Comment:
I don't think this search is required. The previous search on line 344 should itself fail with NPE, right?